locales/en-US.json

```diff
@@ -430,5 +430,12 @@
     "user_created_playlists": "`x` created playlists",
     "user_saved_playlists": "`x` saved playlists",
     "Video unavailable": "Video unavailable",
-    "preferences_save_player_pos_label": "Save playback position: "
+    "preferences_save_player_pos_label": "Save playback position: ",
+    "crash_page_you_found_a_bug": "It looks like you found a bug in Invidious!",
+    "crash_page_before_reporting": "Before reporting a bug, make sure that you have:",
+    "crash_page_refresh": "Tried to refresh the page",
+    "crash_page_switch_instance": "Tried to use another instance",
+    "crash_page_read_the_faq": "looked at the <a href=\"`x`\">Frenquently Asked Queqtions (FAQ)</a>",
```

Comment:
```suggestion
    "crash_page_read_the_faq": "read the <a href=\"`x`\">Frenquently Asked Queqtions (FAQ)</a>",
```

Reply:
Can you re-propose this one? I can't commit it.